website/src/utils/ast.test.js

```diff
@@ -104,3 +104,72 @@ describe("getReactLiveNoInline", () => {
     ).toBe(true);
   });
 });
+
+describe("annotateCodeForPlayground", () => {
+  it("wraps plain jsx code in Fragment", () => {
+    expect(
+      annotateCodeForPlayground("<Button>Apply</Button>").startsWith("<>")
+    ).toBe(true);
+    expect(
+      annotateCodeForPlayground("<Button>Apply</Button>").endsWith("</>;")
+    ).toBe(true);
```

Comment:
IMHO, this feels too verbose. If we test plain JSX, we should state it only once. Having 2 copies is prone to errors. Also, seeing `startsWith` and `endsWith` in these tests make them less readable (harder to scan).
Usually, I wouldn't add helper methods in tests, but I think here the cost of not doing so is too high.
```
function isFragment(code) {
  return /^<>.*<\/>;$/.test(code);
}

it("wraps plain jsx code in Fragment", () => {
  expect(isFragment(annotateCodeForPlayground("<Button>Apply</Button>"))).toBe(true);
});
```

What do you think?

Reply:
Yeah I think it's worth it to add in a helper method seeing as I'm using the `startsWith` and `endsWith` a few times. I generally find regex's pretty hard to read when scanning as well but this one is probably simple enough.